src/guide/essentials/template-refs.md

```diff
@@ -66,34 +67,34 @@ export default {
 ```
 
 </div>
-
-Note that you can only access the ref **after the component is mounted.** If you try to access <span class="options-api">`$refs.input`</span><span class="composition-api">`input`</span> in a template expression, it will be `null` on the first render. This is because the element doesn't exist until after the first render!
+Yodda tuting, siz `ref` ni faqatgina **komponent mounted bo'lgandan keyin** ishlata olasiz. Agar siz <span class="options-api">`$refs.input` ga</span><span class="composition-api">`input` ga</span> template expression ichida bog'lanishga harakat qilsangiz, birinchi render da uning qiymati `null` ga teng bo'ladi. 
+Buning sababi element birinchi renderdan keyin mavjud emas!
 
 <div class="composition-api">
 
-If you are trying to watch the changes of a template ref, make sure to account for the case where the ref has `null` value:
-
+If you are trying to watch the changes of a template ref, refning qiymati `null` bo'lgan holat uchun ta'rif berganingizga ishonch hosil qiling:
```

Comment:
O'zila tarjima qivolila digani bo'sa kere =)

Reply:
![image](https://user-images.githubusercontent.com/12369295/172147148-b39b1720-7f06-42ff-8a06-2f3509042e77.png)

Reply:
😅😅, hozir to'g'rilab qo'yaman

Reply:
To'g'rilandi.